server/eclipse-project/src/main/java/us/freeandfair/corla/query/CountyContestResultQueries.java

```diff
@@ -94,7 +95,23 @@ public static CountyContestResult matching(final County the_county,
     }
     return result;
   }
-  
+
+  /**
+   * Gets CountyContestResults that have the contestName.
+   *
+   * @param contestName The name to match Contest#name.
+   * @return the matching CountyContestResults, or an empty set.
+   */
+  public static List<CountyContestResult> withContestName(final String contestName) {
+    final Session s = Persistence.currentSession();
+    final Query q = s.createQuery("select ccr from CountyContestResult ccr " +
+                                  "inner join Contest c " +
+                                  "on ccr.my_contest = c " +
+                                  "where c.my_name = :contestName");
```

Comment:
:clap: I know it's not type-checked but I far prefer this declarative query style to the "builder" style when not a lot of programmatic control is required.